cli/azd/pkg/infra/provisioning/manager.go

```diff
@@ -122,39 +123,20 @@ func (m *Manager) Destroy(ctx context.Context, deployment *Deployment, options D
 
 // Plans the infrastructure provisioning and orchestrates interactive terminal operations
 func (m *Manager) plan(ctx context.Context) (*DeploymentPlan, error) {
-	var deploymentPlan *DeploymentPlan
 
-	err := m.runAction(
-		ctx,
-		"Planning infrastructure provisioning",
-		m.interactive,
-		func(ctx context.Context, spinner *spin.Spinner) error {
-			planningTask := m.provider.Plan(ctx)
-
-			go func() {
-				for progress := range planningTask.Progress() {
-					m.updateSpinnerTitle(spinner, progress.Message)
-				}
-			}()
-
-			go m.monitorInteraction(spinner, planningTask.Interactive())
-
-			result, err := planningTask.Await()
-			if err != nil {
-				return err
-			}
-
-			deploymentPlan = result
-
-			return nil
-		},
-	)
+	planningTask := m.provider.Plan(ctx)
+	go func() {
+		for progress := range planningTask.Progress() {
+			log.Println(progress.Message)
+		}
+	}()
 
+	deploymentPlan, err := planningTask.Await()
 	if err != nil {
 		return nil, fmt.Errorf("planning infrastructure provisioning: %w", err)
 	}
 
-	m.console.Message(ctx, output.WithSuccessFormat("\nInfrastructure provisioning plan completed successfully"))
+	log.Println("\nInfrastructure provisioning plan completed successfully")
```

Comment:
Does this line provide any value if we're just outputting to the log?

Reply:
I can remove it, sure